terraform/app.tf

```diff
@@ -18,8 +18,9 @@ resource "cloudfoundry_app" "beis-roda-app" {
     "RAILS_ENV"                              = "production"
     "DATA_MIGRATE"                           = var.data_migrate
     "SECRET_KEY_BASE"                        = var.secret_key_base
-    "DOMAIN"                                 = var.domain
-    "ADDITIONAL_DOMAINS"                     = var.additional_domains
+    "CANONICAL_HOSTNAME"                     = var.canonical_hostname
+    "DOMAIN"                                 = "https://${var.canonical_hostname}"
```

Comment:
What are those set to at the moment?

Reply:
I feel like `custom_hostname` should be our canonical hostname too. But I'm not sure if it's set the the right value at the mo, becuase it's stored as a secret.

Reply:
This kinda sums up why I expressly didn't make the new env vars a secret in the actions setup. If something doesn't _need_ to be a secret, it shouldn't be, because we end up with this confusion.